### PR TITLE
op-chain-ops: registry-data tool

### PR DESCRIPTION
**Pull request by protolambda**
**Description**

This is a small tool to convert a `genesis.json` or genesis block header into the assets to be added to the superchain-registry.
See https://github.com/ethereum-optimism/superchain-registry/pull/2

This PR depends on #6581 and #6871 to be merged first.

Fix CLI-4318

### CODE REVIEW

**Comment by protolambda** [comment]
rebased + fixed lint

File: op-chain-ops/cmd/registry-data/main.go [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+			GasUsed:    genesisHeader.GasUsed,
+			ParentHash: genesisHeader.ParentHash,
+			BaseFee:    (*hexutil.Big)(genesisHeader.BaseFee),
+			Alloc:      make(jsonutil.LazySortedJsonMap[common.Address, GenesisAccount]),
+			StateHash:  &genesisHeader.Root,
+		}
+		if err := writeGzipJSON(ctx.Path(OutputFlag.Name), out); err != nil {
+			return fmt.Errorf("failed to write output: %w", err)
+		}
+		return nil
```
Comment (ajsutton) [inline op-chain-ops/cmd/registry-data/main.go:176]: nit: Is it worth making this a separate method?

**Review by ajsutton — APPROVED** [review]
Looks good. Should we have at least a basic test for this?

File: op-chain-ops/cmd/registry-data/main.go (hunk 1 above)
Comment (protolambda) [reply op-chain-ops/cmd/registry-data/main.go:176]: It's only really this tool that does the conversion this way. All the op-node/op-getg code depends onthe data to objects conversion the other way. Maybe when we add new tooling for a superchain-registry CI flow we can consider splitting it out.

**Pull request merged**